stable/artifactory-ha/templates/nginx-deployment.yaml

```diff
@@ -79,6 +79,8 @@ spec:
         volumeMounts:
         - mountPath: {{ .Values.nginx.persistence.mountPath | quote }}
           name: nginx-volume
+        resources:
+{{ toYaml .Values.nginx.resources | indent 10 }}
```

Comment:
maybe "{{ toYaml .Values.initContainers.resources | indent 10 }}" better? this is init container

Reply:
Yes, you're right. Updated it

Reply:
Hi @se-kai , could you please review the PR again? 
Tagging @oumkale and @chukka as well. As soon as this is merged into a release, we can use this as our upstream chart instead of downloading the entire thing into our repo. Thank you 🙏

Reply:
this is already fixed and avialble in current main branch
https://github.com/jfrog/charts/blob/master/stable/artifactory-ha/templates/nginx-deployment.yaml#L79
I used it already and it works fine for me.

Reply:
Hi @se-kai , thank you for the update. Appreciate having resources block added to nginx-deployment. 

We also need `annotations` added to `artifactory-primary-service.yaml`'s `metadata`. It is not originally added and we can't use this chart as an upstream until that's added unfortunately. Can that be taken into account for the next release? 
I can rebase and update this PR with just the `artifactory-primary-service.yaml` changes required if you need.